docs/billing-and-plans/upgrade-from-free-plan.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade from Free Subscription Plan
+intro: Sometimes your projects have a high demand for changes and your Devopness free credits reach the limit. Upgrade from free subscription plan to ensure your team can continue to iterate on the projects.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, in the upper-right corner of any page, click your profile icon then click `Billing & plans`
+    > This page shows the current subscription plan information including: plan name, credit limit, remaining credits and expiration date.
+1. Click `MANAGE YOUR PLAN`
+    > This will open your billing page
```

Comment:
Should be consistent with the wording used in `cancel subscription`.

Reply:
```suggestion
    > This will redirect you to the billing management page
```